ansible/roles/compiled/tasks/common/fftw.yaml

```diff
@@ -0,0 +1,40 @@
+- name: Install fftw
+  vars:
+    fftw_dir: "{{ common_dir }}/fftw"
+    fftw_module_dir: "{{ common_modules }}/fftw"
+    url: "{{ item.url }}"
+    version_number: "{{ item.version_number }}"
+    checksum: "{{ item.checksum }}"
+    install_dir: "{{ fftw_dir }}/{{ version_number }}"
+    unzipdir: "/dev/shm/fftw-{{ version_number }}"
+    build_dir: "{{ unzipdir }}/build"
+    mpi_dir: "{{ item.mpi_dir }}"
+    mpi_module: "{{ item.mpi_module }}"
+  tags:
+    - never
+    - fftw{{ item.version_number }}
+  block:
+    - name: Install fftw {{ version_number }}
+      block:
+        - name: Download, compile and install fftw {{ version_number }}
+          block:
```

Comment:
Again just for my own learning .. the reason for the second block is because of the loop_control that install multiple versions of fftw right?

Reply:
all of them for the same version number.

Reply:
meant different precisions :).

Reply:
Yip, exactly right. Unfortunately you cant loop over a `block` in ansible. I don't know the underlying reason for that, I find it a bit annoying.